### ISSUE TITLE
New checkout fixes/improvments

### ISSUE THREAD

**Issue by julien51** [unlock-protocol/unlock#9106]
# Before launch

- [x] back buttons (tool tips) to go between "steps"
- [X] Arrow on lock selection
- [X] bold/light key price
- [x] missing $ sign on $ price
- [X] price alignment to the right
- [x] Available quantity vs. quantity - too long and will take lot of space
- [X] do not show quantity on the last screen
- [x] What about Sold out state?
- [X] What if the user already has a membership?
- [x] "Buy another" vs "Buy membership"
- [x] Back button on mobile (vs stepper clickable)
- [ ] #9198
- [X] fix transaction hash on "view in explorer"
- [x] if credit card not enabled, and Unlock account: show message about credit card not being available.
- [x] Powered by Unlock in footer

## For later

- [ ] #9199
- [ ] make it more clear that there are multiple options when there are multiple options
- [ ] Add option to pre-select when loading the checkout
- [ ] #9200
- [ ] Make confirmation image customizable?
- [ ] #9201
- [ ] More views for recurring
- [x] show error animation in mint views

**Comment by julien51** [comment]
We need to improve the way "quantities" work. If the lock still has a max number of keys per user to 1 we should not let them configure the quantity (cc @ChialiTsai @searchableguy).

**Comment by searchableguy** [comment]
@julien51 No, that wouldn't work because user can buy multiple memberships for different recipients than themselves. 

Which is why we do not warn them until they enter the recipient and we check if they are allowed to  buy multiple keys for that lock.

**Comment by julien51** [comment]
Good point! Closing!
